### ISSUE TITLE
classification

### ISSUE THREAD

**Issue by orangeslx** [mengzaiqiao/CAN#11]
Hello, perform classification experiments on the embeddings obtained from the 5% validation data set and 10% test data set? It is not written in the paper. Thank you for your reply!

**Comment by mengzaiqiao** [comment]
20% of labeled nodes are used to train an SVM classifier and the rest are used for testing., which has been stated in the paper. We don't have hyper-parameters needed to tune, so we didn't use a validation set.

**Comment by orangeslx** [comment]
Thank you for your reply, you helped me a lot, thank you again. I know what you said. I may not have made it clear. I want to know whether the node embedding data for the classification experiment is obtained by training on the 5% validation data set and 10% test data set?

**Comment by mengzaiqiao** [comment]
@orangeslx Yes. The reported performance of classification in our paper is under such a split. However, you can still train your embeddings using the full datasets, as in this setting, no labels are used for classification and you only use the labels when training the SVM classifier, so it is still a fair comparison.

**Comment by orangeslx** [comment]
您好，我想问的是您论文中分类实验，在进行分类实验的节点嵌入数据，是在5%验证数据集和10%测试数据集下得到的吗？恳求得到您的回复
